client/src/pages/private/ParticipantDetailsView.js

```diff
@@ -280,6 +307,38 @@ export default () => {
               </Grid>
             </Box>
 
+            {/* Participant RoS Info */}
+            <CheckPermissions
+              permittedRoles={['health_authority', 'employer', 'ministry_of_health']}
+            >
+              {participant.ros && (
+                <>
+                  <Box pt={4} pb={2} pl={4} pr={4}>
+                    <Typography variant='h2'>Return of Service</Typography>
+                  </Box>
+
+                  <Box py={2} px={4}>
+                    <Grid className={classes.rootContainer} container spacing={2}>
+                      {Object.keys(rOSKeyMap).map((key) => (
+                        <Grid key={key} item xs={12} sm={6} xl={3}>
+                          <Grid item xs={6}>
```

Comment:
I'm pretty sure we can get rid of inner Grid items.
Item shouldn't contain other sub-items without container. Basically it's container -> item.

Reply:
Check my previous comment.

Reply:
This is a different issue. I'm not talking about having Box/Grid mixed up but rather want to bring attention to incorrect use of `Grid` component